istioctl/cmd/istiodconfig.go

```diff
@@ -377,7 +377,7 @@ func istiodLogCmd() *cobra.Command {
 	outputFormat := "short"
 
 	logCmd := &cobra.Command{
-		Use:   "log [<pod-name>]|[-r|--revision] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[--reset]|[--output|-o short|yaml]",
+		Use:   "log [<pod-name>]|[-r|--revision] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[--reset]|[--output|-o short|json]",
```

Comment:
It should be better to support YAML format instead of revising the example

Reply:
OK, I will try it later, thanks